src/routes/resources/+page.svelte

```diff
@@ -1,96 +1,51 @@
 <script lang="ts">
-	const booksFromPublisher = [
-		{
-			name: 'Svelte 3 Up and Running',
-			link: 'https://www.amazon.com/dp/B08D6T6BKS/',
-			author: 'Alessandro Segala'
-		},
-		{
-			name: 'Svelte and Sapper in Action',
-			link: 'https://www.manning.com/books/svelte-and-sapper-in-action',
-			author: 'R. Mark Volkmann'
-		},
-		{
-			name: 'SvelteKit Up and Running',
-			link: 'https://sveltekitbook.dev',
-			author: 'Dylan Hildenbrand'
-		},
-		{
-			name: 'Svelte with Test-Driven Development',
-			link: 'https://www.amazon.com/dp/1837638330',
-			author: 'Daniel Irvine'
-		}
-	];
-	const booksSelfPublished = [
-		{
-			name: 'Svelte Handbook',
-			link: 'https://flaviocopes.com/page/svelte-handbook/',
-			author: 'Flavio Copes'
-		},
-		{
-			name: 'Simple Svelte',
-			link: 'https://wfq.gumroad.com/l/simple_svelte',
-			author: 'Darren Wang'
-		}
-	];
+	import books from './books.json';
+	import extensions from './extensions.json';
+	import misc from './misc.json';
+	import videos from './videos.json';
 </script>
 
 <div>
 	<h2>Books</h2>
 
-	<div>There are a few books from major publishers:</div>
 	<ul>
-		{#each booksFromPublisher as { name, link, author }}
-			<li><a href={link} target="_blank">{name}</a> by {author}</li>
-		{/each}
-	</ul>
-
-	<div>As well as a couple self-published books:</div>
-	<ul>
-		{#each booksSelfPublished as { name, link, author }}
-			<li><a href={link} target="_blank">{name}</a> by {author}</li>
+		{#each books as { title, url, description }}
+			<li><a href={url} target="_blank">{title}</a> - {description}</li>
 		{/each}
 	</ul>
 
 	<h2>Videos</h2>
 
-	<div>Rich Harris, the creator of Svelte, taught a course:</div>
 	<ul>
-		<li>
-			<a href="https://frontendmasters.com/courses/svelte/" target="_blank">Frontend Masters</a>
-		</li>
+		{#each videos as { title, url, description }}
+			<li><a href={url} target="_blank">{title}</a> - {description}</li>
+		{/each}
 	</ul>
 
-	<div>There are also a number of third-party courses:</div>
-	<ul>
-		<li><a href="https://egghead.io/browse/frameworks/svelte" target="_blank">Egghead</a></li>
-		<li>
-			<a href="https://www.udemy.com/courses/search/?q=sveltejs+svelte" target="_blank">Udemy</a> (Note:
-			Udemy frequently has discounts over 90%)
-		</li>
-		<li><a href="https://www.pluralsight.com/search?q=svelte" target="_blank">Pluralsight</a></li>
-	</ul>
+	<h2>Extensions</h2>
 
-	<div>Finally, there are also YouTube channels and playlists that teach Svelte:</div>
```

Comment:
We lose the distinction between learning platform and YouTube.

Learning platform often have extra bits (documents, projects, support, etc.)

Reply:
Fixed!